app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] as required you should enable the `:confirmable` in order to allow the user to confirm his/her email while signing up

Reply:
### Hi @youmari 
Thank you very much for giving helpful feedback.

Reply:
Fixed.

Reply:
### Hi @DammyShittu @Abenezer-Tilahun 
Thank you very much for approving this project.